### DISCUSSION TITLE
Groups

### DISCUSSION THREAD

**Discussion by aniho91** [mastodon/mastodon#26123] [Q&A]
Does anyone know if Mastodon is planning #groups like on Calckey/Firefish and Twitter respectively?

Friends of mine and I have always worked with closed and moderated groups on certain political topics, brainstorming so to speak. If groups don't come on Mastodon, we'll all have to move to Firefish. But actually we'd like to stay on Mastodon. At the moment we're trying to work through groups -DMs, but you can't thematically delimit them, so it's all a huge mess.

![listen](https://github.com/mastodon/mastodon/assets/132325594/db1104b7-bbe0-4722-9a1a-6fe78326a180)

**Comment by yelvington** [comment]
See discussion extending back a couple of years:
https://github.com/mastodon/mastodon/issues/139

Development work under way:
https://github.com/mastodon/mastodon/pull/19059

External services that implement some group functionality:
https://chirp.social/
https://gup.pe/

**Reply by aniho91** [reply]
Everything that is discussed about groups is about groups where every post is publicly readable, but I am concerned with closed groups.
